types/xar/index.d.ts

```diff
@@ -1,11 +1,11 @@
 // Type definitions for xar 1.1
 // Project: https://github.com/finnp/xar
-// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions by: Florian Imdahl <https://github.com/ffflorian>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node"/>
 
-import { Readable as ReadableStream } from 'stream';
+import { Readable as ReadableStream } from 'readable-stream';
```

Comment:
where is that direct dependency defined in th esource package? what is the gain here? The package (xar) was updated 5 years ago

Reply:
You are right, using `stream` is enough, although `readable-stream` is a core module from Node.js which doesn't need to be defined in the source.

Reply:
I updated my PR.